build.gradle.kts

```diff
@@ -4,6 +4,12 @@ plugins {
 
 group = "com.hivemq.helmcharts"
 
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+    }
+}
+
```

Comment:
was this now intended to be added?

Reply:
@Donnerbart could you please confirm? I can see it was included in your last commit here.
Though from my point of view, it should be fine and good to have as well

Reply:
I've added it here, since we had to remove this setting from the included `hivemq-operator` (because of the Micronaut 3.x Gradle plugin). We have a bunch of Java test classes in this project, I thought it's good to pin them to Java 11.